src/test/java/posters/cucumber/support/ProductSupport.java

```diff
@@ -3,19 +3,19 @@
 import io.cucumber.java.en.Then;
 import io.cucumber.java.en.When;
 import posters.pageobjects.pages.browsing.CategoryPage;
-import posters.pageobjects.pages.browsing.ProductdetailPage;
+import posters.pageobjects.pages.browsing.ProductDetailPage;
 
 public class ProductSupport
 {
     @When("^product \"([^\"]*)\" is opened$")
-    public ProductdetailPage clickProductByName(String productName)
+    public ProductDetailPage clickProductByName(String productName)
```

Comment:
@oomelianchuk is a return type needed at this place?
In general the Steps are not able to communicate with each other on this level, so please remove the return if it'n unused.

Reply:
I couldn't find a reason, why we would need this. Removed